tests/monitoring/component_monitoring.go

```diff
@@ -236,7 +236,7 @@ var _ = Describe("[Serial][sig-monitoring]Component Monitoring", Serial, decorat
 			}, 5*time.Minute, 500*time.Millisecond).Should(Succeed())
 		})
 
-		It("VirtHandlerRESTErrorsBurst and VirtHandlerRESTErrorsHigh should be triggered when requests to virt-handler are failing", func() {
+		PIt("VirtHandlerRESTErrorsBurst and VirtHandlerRESTErrorsHigh should be triggered when requests to virt-handler are failing", func() {
```

Comment:
I found the E2E tests for virt-handler were having the same flakiness as https://github.com/kubevirt/kubevirt/pull/11601/commits/415964c6d985a852110f52a6fbe99ffd02533a61#diff-5973bd8e1862c32cabfb41df6b9f6ebd8c81eb221b43965d12779f2476514579R224

I also marked it as pending to check them later together

Reply:
@machadovilaca Does the flakiness related to this PR changes?

Reply:
yeah, not really, different PR?

Reply:
Yes, it's better to split so we can track these changes better.